dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/DateUtils.java

```diff
@@ -575,6 +575,12 @@ public static long getRemainTime(Date baseTime, long intervalSeconds) {
             return 0;
         }
         long usedTime = (System.currentTimeMillis() - baseTime.getTime()) / 1000;
+        // ignore worker's clock later than master nodes.
```

Comment:
> It's better don't add this in this method, this is Utils. And I don't think we need to deal with this, we only need to announce the clock should sync in master/worker, since this change don't solve the problem when the clock is not sync.

yes, i agree with u. I also agree that there is no good way to deal with clock inconsistencies between clusters. just add doc should be enough.